hooks/hookexecution/executor_test.go

```diff
@@ -861,6 +1147,23 @@ func (e TestWithTimeoutPlanBuilder) PlanForRawAuctionStage(_ string, _ *config.A
 	}
 }
 
+func (e TestWithTimeoutPlanBuilder) PlanForProcessedAuctionStage(_ string, _ *config.Account) hooks.Plan[hookstage.ProcessedAuctionRequest] {
```

Comment:
Again for my own understanding, why are there parameters that are defined for this function, but are unused in the function?

Reply:
Because this is kind of a mocker for the PlanBuilder where we immediately return a ready execution plan without the need to actually parse the config, register all hooks in the repo and other heavy things not related to the actual purpose of the tests for the hook executor. All those things are covered with their own unit tests in `hooks/plan_test.go`